src/main/java/com/minecolonies/coremod/colony/managers/EventStructureManager.java

```diff
@@ -156,8 +157,16 @@ public void loadBackupForEvent(final int eventID)
 
             if (entry.getValue() == eventID)
             {
-                final String backupPath = String.valueOf(colony.getID()) + colony.getDimension() + entry.getKey();
-                final String fileName = new StructureName("cache", "backup", Structures.SCHEMATICS_PREFIX + STRUCTURE_BACKUP_FOLDER).toString() + backupPath;
+                //TODO: remove compat for colony.getDimension()-based file names after sufficient time has passed from PR#6305
+                final String oldBackupPath = String.valueOf(colony.getID()) + colony.getDimension() + entry.getKey();
+                String fileName = new StructureName("cache", "backup", Structures.SCHEMATICS_PREFIX + STRUCTURE_BACKUP_FOLDER).toString() + oldBackupPath;
+
+                File blueprintFile = new File(fileName);
+                if(!blueprintFile.exists())
```

Comment:
should be the other way around, first new backup path, if not exist try old

Reply:
Ok.  Swapped to that approach, and also had it use the Structurize handlers for the check.